pom.xml

```diff
@@ -12,12 +12,14 @@
         <maven.compiler.source>21</maven.compiler.source>
         <maven.compiler.target>21</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <tinkar-core.version>1.67.0</tinkar-core.version>
+        <tinkar-core.version>1.71.0-SNAPSHOT</tinkar-core.version>
         <tinkar-schema.version>1.27.0</tinkar-schema.version>
         <tinkar-transformer-api.version>1.7.0</tinkar-transformer-api.version>
         <java-uuid-generator.version>4.3.0</java-uuid-generator.version>
         <jackson-databind.version>2.14.2</jackson-databind.version>
         <slf4j.version>2.0.13</slf4j.version>
+        <tinkar-composer.version>1.4.0-SNAPSHOT</tinkar-composer.version>
```

Comment:
Is there a reason you're using 1.4.0 of composer? We shouldn't refer to snapshot versions if we can help it. Can you revert this to version 1.3.0?